### PR TITLE
build: sending emails

### PR DESCRIPTION
**Pull request by AyaQunoo**

### CODE REVIEW

File: server/utils/email/nodemailer.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const nodemailer = require('nodemailer');
+
+const user = process.env.USER;
```
Comment (devstm) [inline server/utils/email/nodemailer.js]: use app.config or mail.config 
const user = process.env.USER;

File: server/utils/email/nodemailer.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+const pass = process.env.PASSWORD;
+const port = process.env.PORT_STMP;
+const sendEmail = async (to, subject, message, text) => {
+  const transporter = nodemailer.createTransport({
+    host: 'smtp.gmail.com',
+    port,
+    secure: false,
+    auth: {
+      user,
```
Comment (devstm) [inline server/utils/email/nodemailer.js:14]: not descripteve names

File: server/utils/email/nodemailer.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      pass,
+    },
+  });
+  await transporter.sendMail({
+    from: user,
```
Comment (devstm) [inline server/utils/email/nodemailer.js:19]: from: "noreplay@fluento.ps"

**Review by devstm — CHANGES_REQUESTED** [review]

**Review by devstm — APPROVED** [review]

**Review by abdullah-alsharif — APPROVED** [review]
solve Conflict

**Pull request merged**